app/build.gradle

```diff
@@ -62,9 +62,17 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             resValue("bool", "FIREBASE_ANALYTICS_DEACTIVATED", "false")
         }
+        // dkh - 5/3/21  - These changes came  from Chris Hubbard (pull request 568)
+        // Updated applicationIdSuffix to allow debug versions of app crashes to be captured separate from
+        // continuous or product builds in Firebase.  Also, make sure we capture analytics by
```

Comment:
crashes in Continuous or Publish builds.

Reply:
Updated comment